k8s/prod/common/ethos/repl-docmosis-service.yaml

```diff
@@ -24,6 +24,7 @@ spec:
       environment:
         TORNADO_URL: https://docmosis.platform.hmcts.net/rs/render
         IDAM_API_URL: https://idam-api.platform.hmcts.net
+        CCD_GATEWAY_BASE_URL: https://gateway.ccd.platform.hmcts.net
```

Comment:
also bump your chart version to 0.1.6 https://github.com/hmcts/ethos-repl-docmosis-service/blob/master/charts/ethos-repl-docmosis-backend/Chart.yaml